Game/GDKTestGyms.uproject

```diff
@@ -14,6 +14,10 @@
 		}
 	],
 	"Plugins": [
+		{
+			"Name": "GameplayAbilities",
```

Comment:
I'm not sure if this is needed - the UnrealGDK plugin isn't listed here either but still used.